pkg/controller/jmeter_integration_test.go

```diff
@@ -34,6 +34,7 @@ func TestMain(m *testing.M) {
 }
 
 func TestIntegrationJMeter(t *testing.T) {
+	t.Skip("Skipping JMeter integration tests")
```

Comment:
dev leftover?

Reply:
Because of current action limits, we can't run JMeter tests (too heavy), that is why we replaced it by Kangal fake provider tests to test Kangal controller logic only.